float64[] joint_angles
---
bool homed